### ISSUE TITLE
ImageNet Evaluation for EfficientNet-edgeTPU models have num classes inconsistent with the rest of the model. 

### ISSUE THREAD

**Issue by sarahmass** [tensorflow/tpu#594]
I am working on evaluation EfficientNet-edge TPU models on my own machine.  And when I ran through the eval_ckpt_exampl with efficientNetLabels_map.txt which should be run with include_background_label=False  I get predicted class offset by one.  See results below:

![image](https://user-images.githubusercontent.com/54375901/68413276-ca328800-0142-11ea-8f0e-26855fca6ed7.png)

I believe because the edgeTPU models were trained with 1001 classes which includes the background class but the base model runs with 1000 classes.  I can make it work correctly if I incorrectly set the include_background_label=True.  See output below:
![image](https://user-images.githubusercontent.com/54375901/68413547-52189200-0143-11ea-865f-f2457cbb0147.png)

Because I am running inference using a pretrained model I can not just change the number of classes in the efficientNet-edgeTPU model because then the model parameters will have dimension mismatch.  

In the code  imageNet_input.py uses 1000 classes and if you have include_background_label=True it adds 1 to the number of classes.  And in dataset_parser you have this:
![image](https://user-images.githubusercontent.com/54375901/68414211-9b1d1600-0144-11ea-91f3-0941790eb860.png)

This is where the offset in predicted class occurs.  I do not have a labels_map in the correct format to check to see what happens when I use a map with the background label.  I would have to create one, and my work around is faster right now.  So, I will continue to use include_background_label=True.   To work around this issue.
